tools/checked-c-convert/ProgramInfo.cpp

```diff
@@ -1042,22 +1052,25 @@ bool ProgramInfo::getDeclStmtForDecl(Decl *D, DeclStmt *&St) {
 // currentVariable field of V is that constraint variable. Returns false if
 // a constraint variable cannot be found.
 std::set<ConstraintVariable *> 
-ProgramInfo::getVariableHelper(Expr *E, 
-  std::set<ConstraintVariable *> V, ASTContext *C) {
+ProgramInfo::getVariableHelper( Expr                            *E, 
```

Comment:
The formatting seems slight off - there seems to be an extra space.

Reply:
Could you update the comment to describe what the ifc parameter means?

Reply:
I added a longer comment at the declaration of `getVariable` that explains the purpose of the flag, and then cited that at `getVariableHelper`.